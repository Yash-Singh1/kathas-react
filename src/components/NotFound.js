import React from 'react';
import { Link } from 'react-router-dom';
import NavigationBar from './NavigationBar';

const NotFound = () => (
  <NavigationBar>
    <h1>404: Not Found</h1>
    <p>
      The path that you requested was not found. Try going back to the{' '}
      <Link to='/'>homepage</Link>.
    </p>
  </NavigationBar>
);

export default NotFound;
