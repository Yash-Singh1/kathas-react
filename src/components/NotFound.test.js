import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  const notFound = shallow(<NotFound />);

  it('renders properly', () => {
    expect(notFound).toMatchSnapshot();
  });
});
