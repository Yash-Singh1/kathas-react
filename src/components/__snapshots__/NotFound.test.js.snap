// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders properly 1`] = `
<NavigationBar>
  <h1>
    404: Not Found
  </h1>
  <p>
    The path that you requested was not found. Try going back to the
     
    <Link
      to="/"
    >
      homepage
    </Link>
    .
  </p>
</NavigationBar>
`;
